### ISSUE TITLE
Broken visibility of categories

### ISSUE THREAD

**Issue by dondolarson** [olsh/Feedly-Notifier#170]
**Describe the bug**
I've "update only following categories" setting unchecked and one of my categories won't notify me about new posts, and it's even not visible in the category tab. Never seen it and was wondering all the time why I'm not receiving any notifications from it. There's about 10 unread notifications for this category when I go straight to Feedly webpage.

**Desktop (please complete the following information):**
 - OS: Windows 10 1909
 - Browser: Chrome 
 - Version: 84.0.4147.105

**Comment by olsh** [comment]
Hi @dondolarson ,

Thanks for the reporting.
Have you tried to re-login? Are there any errors when you go to the options page? Does the category have some special characters?

**Comment by dondolarson** [comment]
Yes, I did. I've cleaned browser cache too. No one of them has special characters and I can't see any errors either.

**Comment by olsh** [comment]
@don-dolarson — sorry this went quiet. Does it still happen on a current version? A lot has changed since 2020, including a rewrite to Manifest V3 and, in #385, a fix for the subscription list being cached and refetched incorrectly, which is one plausible way a category could go missing from the list.

If it does still happen, two things would help narrow it down:

1. On the options page, does the missing category appear in the category list under Filters? That tells us whether it is missing from the subscription data or only from the popup's tabs.
2. Roughly how many categories and subscriptions does the account have? The extension fetches a bounded number of articles per update, so a very large account behaves differently from a small one.

Note this is specifically about a category that is **entirely invisible**. A category that shows only *some* of its articles is the "max number of feeds" cap, which is #134.

Closing this in a while if there is no reply — please just say so and it will stay open.
